src/index.html

```diff
@@ -16,9 +18,9 @@ <h1 class="fw-bold my-3">Leaderboard</h1>
         <section id="recent-scores" class="col-md-6 my-3">
             <div class="d-flex justify-content-around my-3">
                 <h2 class="fw-bold">Recent scores</h2>
-                <button class="btn btn-lg btn-outline-success">Refresh</button>
+                <button id="refresh-score" class="btn btn-lg btn-outline-success">Refresh</button>
             </div>
-            <ul class="list-group my-3">
+            <ul id="list-scores" class="list-group my-3">
```

Comment:
- [ Required ] It would be better if the list of scores is only populated by data fetched from the `API` when the page is loaded. Currently, when I open the page the scores loaded on the page are hardcoded on the page. When the page is refreshed or loaded the scores on the page should be those `fetched` from the `API`

## Video
https://user-images.githubusercontent.com/16392046/216002431-6fc274fd-b701-4a94-8347-319651fee04b.mp4